uint32 session_id
uint32 camera_id
uint32 cuser_id
uint32 face_id
time ts
sensor_msgs/Image thumb
